### ISSUE TITLE
Do You have the pretrained model？？？

### ISSUE THREAD

**Issue by fanbinqi** [timctho/shufflenet-v2-tensorflow#2]

**Comment by codecolony** [comment]
Would like to know the same too.

**Comment by ppwwyyxx** [comment]
I have reproduced the ImageNet results in the paper and uploaded the pre-trained models here: https://github.com/tensorpack/tensorpack/tree/master/examples/ImageNetModels
